deploy/templates/nstemplatetiers/appstudio-env/cluster.yaml

```diff
@@ -40,9 +42,10 @@ objects:
         count/ingresses.extensions: "30"
         count/routes.route.openshift.io: "30"
     selector:
-      annotations:
-        openshift.io/requester: ${SPACE_NAME}
-      labels: null
+      annotations: null
```

Comment:
do we need to explicitely use `annotations: null` or could we just remove this kind of line?

Reply:
I was wondering the same, but all the examples I've found have one of the two configs set to `nul`: https://docs.openshift.com/container-platform/3.11/admin_guide/multiproject_quota.html

so I just left it as is.

Reply:
I can remove it , and see if tests are still passing (I'm almost sure they will). It's just cosmetics , since previously the `labels` key was set to null as well.

Reply:
yeah, it's cosmetics so no need to waste time on this if you don't want to! (my apologies, I did not mean to stick on this particular comment)